GVFS/GVFS/CommandLine/UpgradeVerb.cs

```diff
@@ -214,7 +249,12 @@ private bool TryCopyUpgradeTool(out string upgraderExePath, out string consoleEr
 
             using (ITracer activity = this.tracer.StartActivity(nameof(this.TryCopyUpgradeTool), EventLevel.Informational))
             {
-                if (!this.upgrader.TrySetupToolsDirectory(out upgraderExePath, out consoleError))
+                if (this.DryRun)

```

Comment:
Why do we not copy the upgrade to the alternate directory in dry-run mode? Could this mask an issue during functional tests?

Reply:
yes possible...good catch.